README.adoc

```diff
@@ -106,6 +116,20 @@ TI-Messenger Dokumentation
 └── link:ReleaseNotes.md[ReleaseNotes.md]
 ----
 
+== Quellen
+Die nachfolgende Tabelle enthält die in der vorliegenden Online Dokumentation referenzierten Dokumente der gematik. 
+
+|===
+|[Quelle] |Herausgeber: Titel
+
+|link:https://fachportal.gematik.de/fachportal-import/files/gemSpec_TI-Messenger-Dienst_V1.1.1.pdf[gemSpec_TI-Messenger-Dienst_v1.1.1] |gematik: Spezifikation TI-Messenger-Dienst
+|link:https://fachportal.gematik.de/fachportal-import/files/gemSpec_TI-Messenger-FD_V1.1.1.pdf[gemSpec_TI-Messenger-FD_v1.1.1] |gematik: Spezifikation TI-Messenger-Fachdienst
+|link:https://fachportal.gematik.de/fachportal-import/files/gemSpec_TI-Messenger-Client_V1.1.1.pdf[gemSpec_TI-Messenger-Client_v1.1.1] |gematik: Spezifikation TI-Messenger-Client
+|link:https://gemspec.gematik.de/docs/gemSpec/gemSpec_VZD_FHIR_Directory/gemSpec_VZD_FHIR_Directory_V1.3.0[gemSpec_VZD_FHIR_Directory_v1.3.0] |gematik: Spezifikation Verzeichnisdienst FHIR-Directory
+|link:https://fachportal.gematik.de/hersteller-anbieter/komponenten-dienste/identity-provider-idp[zentraler IDP_Dienst] |gematik: zentraler IDP-Dienst der gematik
+|link:https://fachportal.gematik.de/hersteller-anbieter/komponenten-dienste/authenticator[Authenticator] |gematik: Authenticator der gematik
+|===
+
```

Comment:
Hier bin ich mir nicht ganz sicher was wir machen sollen. Zumindest die Links zu den alten Specs sollten denke ich verschwinden. Eventuell könnten wir uns die anderen Links auch sparen weil man sie ja notfalls auch in den neuen Specs findet?